### PR TITLE
[chassis-packet] minigraph parsing and BGP template changes

### PR DESCRIPTION
**Pull request by abdosi**
What I did:

 ```
 Initial Commit for Chassis in Packet mode and multi-asic.

    •       Internal BGP Peering between LCx-ASICy is done using Loopback4096
    •       Internal BGP Peer will be classified as BGP_INTERNAL_NEIGHBOR with source updated as Loopback4096
    *       Graph attribute <VOQChassisInternal> change to <ChassisInternal> and taking switch_type as value instead of true/false
    •       DEVICE_METADATA will have “switch_type” define as “chassis-packet”.  We already have “voq” define. This will be used in BGP template.
    •       Static Routes are define using SONiC schema.
    *       Fixed Vlan Sub Interface Parsing in Graph when configured on Port-Channel.
```

How I Verify:

- Added unit test case for graph and BGP Template changes
- Manual Verification on the DUT.

### CODE REVIEW

**Comment by abdosi** [comment]
@anamehra Please review from your side also.

File: src/sonic-config-engine/tests/test_cfggen.py [hunk 1]
```diff
@@ -831,5 +833,28 @@ def test_minigraph_dhcp(self):
...
+       
+    def test_minigraph_bgp_packet_chassis_peer(self):
+        argument = '-m "' + self.packet_chassis_graph + '" -p "' + self.packet_chassis_port_ini + '" -n "' + "asic1" + '" -v "BGP_INTERNAL_NEIGHBOR[\'8.0.0.1\']"'
+        output = self.run_script(argument)
+        self.assertEqual(
+            utils.to_dict(output.strip()),
+            utils.to_dict("{'rrclient': 0, 'name': 'str2-8808-lc0-ASIC1', 'local_addr': '8.0.0.3', 'nhopself': 0, 'admin_status': 'up', 'holdtime': '0', 'asn': '65100', 'keepalive': '0'}")
+        )
+
+    def test_minigraph_bgp_packet_chassis_static_route(self):
```
Comment (arlakshm) [inline src/sonic-config-engine/tests/test_cfggen.py:845]: add a tests for the single asic linecard?

File: src/sonic-config-engine/tests/sample-chassis-packet-lc-port-config.ini [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# name lanes alias index speed asic_port_name role source_npu target_npu
```
[the remark this answers is not in the document]
Comment (arlakshm) [reply src/sonic-config-engine/tests/sample-chassis-packet-lc-port-config.ini:1]: Fix the file formatting as well

File: src/sonic-config-engine/minigraph.py [hunk 3]
```diff
@@ -1394,12 +1405,12 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
...
-        elif intf[0][0:11] == 'PortChannel':
-            pc_intfs[intf] = {}
-            pc_intfs[intf[0]] = {}
         elif VLAN_SUB_INTERFACE_SEPARATOR in intf[0]:
             vlan_sub_intfs[intf] = {}
             vlan_sub_intfs[intf[0]] = {'admin_status': 'up'}
+        elif intf[0][0:11] == 'PortChannel':
+            pc_intfs[intf] = {}
+            pc_intfs[intf[0]] = {}
         else:
```
[the remark this answers is not in the document]
Comment (abdosi) [reply src/sonic-config-engine/minigraph.py:1414]: yes for PortChannel Vlan SubInterface. Parsing is done by existing code: https://github.com/Azure/sonic-buildimage/blob/6e2fecfa6b8be3cf2605c6673438339d20902a32/src/sonic-config-engine/minigraph.py#L448

File: src/sonic-config-engine/tests/sample-chassis-packet-lc-port-config.ini (hunk 2 above)
[the remark this answers is not in the document]
Comment (abdosi) [reply src/sonic-config-engine/tests/sample-chassis-packet-lc-port-config.ini:1]: working on another PR.. will fix it there.

**Pull request merged**